week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -1,25 +1,61 @@
 class MusicPlayer {
-    // Add your code here
-
+  constructor() {
+    this.playlist = [];
+    this.playingItem = undefined;
+  }
+  add(song, artist) {
+    let newTrack = {};
+    newTrack.artist = artist;
+    newTrack.song = song;
+    this.playlist.push(newTrack);
+  }
+  play() {
+    if (this.playlist.length != 0) {
+      if (this.playingItem == undefined) {
+        this.playingItem = 0;
+      }
```

Comment:
Is there any reason to not directly initialise `playingItem` to `0` in the `constructor`?